src/setuptools_betterproto/_command.py

```diff
@@ -113,3 +116,50 @@ def run(self) -> None:
 
         _logger.info("compiling proto files via: %s", " ".join(protoc_cmd))
         subprocess.run(protoc_cmd, check=True)
+
+
+class AddProtoFiles(BaseProtoCommand):
+    """A command to add the proto files to the source distribution."""
+
+    def run(self) -> None:
+        """Copy the proto files to the source distribution."""
+        proto_files = self.config.expanded_proto_files
+        include_files = self.config.expanded_include_files
+
+        if include_files and not proto_files:
+            _logger.warning(
+                "Some proto files (%s) were found in the `include_paths` (%s), but "
+                "no proto files where found in the `proto_path` (%s). We are not adding "
+                "the found include files to the source distribution automatically!",
+                len(include_files),
+                ", ".join(self.config.include_paths),
+                self.config.proto_path,
+            )
+
+        if not proto_files:
```

Comment:
Is this case common or does it denote a problem?

Reply:
It would be weird that you use this setuptools plugin and you don't have proto files, but a warning will be shown when compiling the files at a later stage. This is just a hack to overcome the issue with the file discovery. I will add a comment to clarify.